src/ontology/mondo-edit.obo

```diff
@@ -434336,6 +434331,28 @@ property_value: confidence "0.33333333333333304" xsd:double
 property_value: seeAlso https://rarediseases.info.nih.gov/diseases/2844/gtp-cyclohydrolase-i-deficiency xsd:anyURI {source="GARD:0002844"}
 created_by: http://orcid.org/0000-0001-5208-3432
 
+[Term]
+id: MONDO:0100188
+name: combined ApoA-I and ApoC-III deficiency
+def: "A hypoalphalipoproteinemia that is has its basis in the disruption of ApoA-I and ApoC-III." [MONDO:patterns/inborn_metabolic_disrupts, PMID:3118360]
+synonym: "ApoA-I and apoC-III deficiency, combined" EXACT [OMIM:618463]
+is_a: MONDO:0017773 {source="PMID:3118360"} ! hypoalphalipoproteinemia
+relationship: disease_has_basis_in_dysfunction_of http://identifiers.org/hgnc/600 {source="OMIM:618463"} ! APOA1
+relationship: disease_has_basis_in_dysfunction_of http://identifiers.org/hgnc/610 {source="PMID:3118360"} ! APOC3
+created_by: http://orcid.org/0000-0001-5208-3432
+
+[Term]
+id: MONDO:0100189
+name: apolipoprotein A-I deficiency
+def: "A rare lipoprotein metabolism disorder characterized biochemically by complete absence of apolipoprotein AI and extremely low plasma high density lipoprotein (HDL) cholesterol, and clinically by corneal opacities and xanthomas complicated with premature coronary heart disease (CHD)." [Orphanet:425]
+synonym: "ApoA-I deficiency" EXACT [Orphanet:425]
+synonym: "familial apoA-I deficiency" EXACT [Orphanet:425]
+synonym: "familial hypoalphalipoproteinemia" EXACT [Orphanet:425]
+is_a: MONDO:0017773 ! hypoalphalipoproteinemia
+relationship: disease_has_basis_in_dysfunction_of http://identifiers.org/hgnc/29 {xref="Orphanet:425"} ! ABCA1
```

Comment:
is this true? What's the evidence?

Remember OWL is monotonic, ORDO is not using OWL correctly, they will list multiple genes their intent is these are interpreted as ANDs. We need to add this to docs @matentzn

Reply:
this is from the Orphanet page:
https://www.orpha.net/consor/cgi-bin/Disease_Genes_Simple.php?lng=EN&LnkId=2927&Typ=Pat&diseaseType=Gen&from=rightMenu

how should this be captured in Mondo?
should it be 'disease has basis in dysfunction of' some ABC1 and APOA1 
?

Reply:
add source: Orphanet:425